src/software/thunderscope/thunderscope_main.py

```diff
@@ -155,12 +155,6 @@
         action="store_true",
         help="show pass cost visualization layer",
     )
-    parser.add_argument(
-        "--disable_estop",
-        action="store_true",
-        default=False,
-        help="Disables checking for estop plugged in (ONLY USE FOR LOCAL TESTING)",
-    )
```

Comment:
replace this for a flag for software estop

Reply:
added a flag for this but forgot to push lol